pkg/providers/instancetype/zz_generated.bandwidth.go

```diff
@@ -20,56 +20,46 @@ package instancetype
 
 var (
 	InstanceTypeBandwidthMegabits = map[string]int64{
-		// f1.2xlarge is not available in https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-network-bandwidth.html
-		// g3.16xlarge is not available in https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-network-bandwidth.html
-		"t2.nano":           32,
+		// c3.large is not available in https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-network-bandwidth.html
```

Comment:
@bwagner5 Do we know why we lost the bandwidth details for these instance types?

Reply:
The question I had was why they were there in the first place. They are wrong as is, so this PR corrects that.

Reply:
Still unclear how these were added. Looks like they've been that way for the entire time this file has existed: https://github.com/aws/karpenter/pull/3432